app/import/process-backlog.js

```diff
@@ -0,0 +1,74 @@
+const sequelize = require('../config/db')
+const { v4: uuidv4 } = require('uuid')
+const getBreed = require('../lookups/dog-breed')
+// const getMicrochipType = require('../lookups/microchip-type')
+const importSchema = require('./imported-dog-schema')
+
+const process = async (maxRecords) => {
+  maxRecords = maxRecords || 99999
+  let rowsProcessed = 0
+  let rowsInError = 0
+  let rowsIntoDb = 0
+
+  const xlBullyBreedId = (await getBreed('XL Bully')).dataValues.id
```

Comment:
This will need to lookup the breed of the current dog record, rather than being hard coded to XL Bully.

There shouldn't be any XL Bully breeds in the Access DB.

Reply:
Haha - yes, good spot. That was left in by mistake fro my testing as I didn't have the full list of breeds. I'll amend that